### PR TITLE
refactor: refactor models/multisig with lumos

### PR DESCRIPTION
**Pull request by twhy**

### CODE REVIEW

**Comment by twhy** [comment]
@Keith-CY Please trigger workflows and review this when you're available.

File: packages/neuron-wallet/src/models/multisig.ts [hunk 1]
```diff
@@ -9,9 +11,11 @@ export interface MultisigPrefix {
   N: string
 }
 
+const BLAKE160_HEX_LENGTH = 42
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/models/multisig.ts:14]: `BLAKE160_HEX_LENGTH` can be exported in `./blake2b`

**Review by Keith-CY — APPROVED** [review]

File: packages/neuron-wallet/src/models/multisig.ts (hunk 1 above)
Comment (twhy) [reply packages/neuron-wallet/src/models/multisig.ts]: OK. Done!

**Review by yanguoyu — APPROVED** [review]

**Comment by twhy** [comment]
The original commit was signed but maybe the rebase made it unsigned.
image

**Review by homura — APPROVED** [review]

**Comment by Keith-CY** [comment]
> The original commit was signed but maybe the rebase made it unsigned. image

Feel free to force-push to override the unsigned commit

**Comment by twhy** [comment]
@Keith-CY  This PR can be merged now, maybe run the workflows once more.

**Pull request merged**